docs/http_api.rst

```diff
@@ -298,6 +298,10 @@ to belong to exactly one organization, but an organization can have many teams.
 
     Get a list of existing organizations
 
+    :queryparam archived:
+        (optional) If true, shows archived organizations. If false, shows
+        organizations that are not archived. Defaults to false.
```

Comment:
I actually just realised that the way we are doing it is incorrect, since `bool("false") == True`. So I'm thinking we make the options `true`, `false`, or `both`?